android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -51,9 +51,13 @@
     private static PackageInfo packageInfo;
     private SentryOptions sentryOptions;
 
+  private long nativeStartedTime;
+
     public RNSentryModule(ReactApplicationContext reactContext) {
         super(reactContext);
         RNSentryModule.packageInfo = getPackageInfo(reactContext);
+
+    nativeStartedTime = System.currentTimeMillis();
```

Comment:
that's too late, maybe you should read the app start up from the Android SDK, but its not merged yet

Reply:
this also does not account for warm/cold start up

Reply:
Oh this is just a proof of concept of how JS would handle it, these numbers would be taken from the native SDK